main/domoticz.cpp

```diff
@@ -374,6 +374,19 @@ void daemonize(const char *rundir, const char *pidfile)
 		sysctl(mib, 4, pathName, &cb, NULL, 0);
 		return cb;
 	}
+#elif defined(__NetBSD__)
```

Comment:
We could join this with FreeBSD and ifdef `mib[...]` part.

Reply:
Less duplication - I agree.